angular/projects/admin-nrpti/src/app/records/orders/order-lng-detail/order-lng-detail.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
+import { OrderLNG } from '../../../../../../common/src/app/models/lng/order-lng';
+import { Subject } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RecordComponent } from '../../utils/record-component';
+import { DatePipe } from '@angular/common';
+import { FactoryService } from '../../../services/factory.service';
+
+@Component({
+  selector: 'app-order-lng-detail',
+  templateUrl: './order-lng-detail.component.html',
+  styleUrls: ['./order-lng-detail.component.scss']
+})
+export class OrderLNGDetailComponent extends RecordComponent implements OnInit, OnDestroy {
+  private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
+
+  public isLoading = true;
+  public isPublishing = false;
```

Comment:
This is one of many places where isPublishing is not used in the UI at all. Should be tossed.

Reply:
Removed.